src/scss/pages/_reservation.scss

```diff
@@ -6,8 +6,11 @@
         font-size: .875rem;
         font-weight: var(--weight-600);
         @extend %text-orange;
+
         &:hover {
```

Comment:
please add :focus as well in this line .so when user focus see the chagnes

Reply:
okay i will do that